test/python/datadog_checks/base/checks/__init__.py

```diff
@@ -0,0 +1,34 @@
+import sys
+
+
+# AgentCheck stubs for testing
+class __AgentCheckPy3(object):
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def run(self):
+        return "result"
+
+    @staticmethod
+    def load_config(yaml_str):
+        pass
+
+
+class __AgentCheckPy2(object):
+    def __init__(self, *args, **kwargs):
+        pass
+
+    def run(self):
+        return "result"
+
+    @classmethod
```

Comment:
staticmethod, there was an error in the original check being addressed here https://github.com/DataDog/integrations-core/pull/3127

Reply:
Sure!